### PR TITLE
Remove the strict puppet-foreman requirement

### PR DESCRIPTION
**Pull request by ekohl**
This amends ff942a72bf1918fa42bea77e82f7c1d7d83e69ef / https://github.com/theforeman/puppet-puppet/issues/154 which made the code change.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,21 @@ Standalone agent with defaults:
 
     echo include puppet | puppet --modulepath /path_to/extracted_tarball
 
+# Integration with foreman
+
+With the 3.0.0 release the foreman integration became optional. Note that you
```
Comment (domcleal) [inline README.md]: *Foreman

File: README.md [hunk 2]
```diff
@@ -30,6 +30,21 @@ Standalone agent with defaults:
...
+need the [puppet-foreman module](https://github.com/theforeman/puppet-foreman)
+installed.
+
+It will still by default install the foreman integration when server is true,
```
Comment (domcleal) [inline README.md]: ditto

File: README.md [hunk 3]
```diff
@@ -64,13 +79,6 @@ wrapper classes or even your ENC (if it supports param classes). For example:
       server_post_hook_content => 'puppetserver/post-hook.puppet',
     }
 
-    # Perhaps you want to install without foreman?
-    class { '::puppet':
-      server                => true,
-      server_reports        => 'store',
```
Comment (domcleal) [inline README.md]: Do you need to keep these params in the above example?

Comment (ekohl) [inline README.md]: You're right, we still default to ```foreman``` there.

File: README.md (hunk 2 above)
Comment (ekohl) [inline README.md]: You mean literal with backticks? Isn't that a github flavor instead of plain markdown?

**Comment by ekohl** [comment]
Amended

**Comment by domcleal** [comment]
Thanks!

**Pull request closed without merging**